eclipse-collections-code-generator-maven-plugin/pom.xml

```diff
@@ -97,6 +97,11 @@
                 <artifactId>maven-enforcer-plugin</artifactId>
             </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-plugin-plugin</artifactId>
+                <version>3.5</version>
```

Comment:
Just noticed this in the plugins section. Shouldn't it go in pluginManagement for consistency? Seriously asking - not sure.